src/Gui/DlgPreferencesImp.cpp

```diff
@@ -204,11 +220,38 @@ void DlgPreferencesImp::createPageInGroup(QTabWidget *tabWidget, const std::stri
     }
 }
 
+void DlgPreferencesImp::relabelResetButtons()
+{
+    int groupIndex = ui->listBox->currentRow();
+    QFontMetrics fm(font());
+    QString group = fm.elidedText(ui->listBox->item(groupIndex)->text(), Qt::ElideRight, ui->buttonResetGroup->width()-4);
+    
+    QTabWidget* tabWidget = static_cast<QTabWidget*>(ui->tabWidgetStack->currentWidget());
+    int tabIndex = tabWidget->currentIndex();
+
+
+    ui->buttonResetGroup->setText(tr(std::string("Reset Group\n" + group.toStdString()).c_str()));
+    ui->buttonResetTab->setText(tr(std::string("Reset Tab " + tabWidget->tabText(tabIndex).toStdString()).c_str()));

```

Comment:
What you need here is to use the String `arg` function to construct your string, so that translators see the right thing. It will looks like `tr("Reset Group %1").arg(groupName)`. I wouldn't include the "\n" either, that will only work in English. So tell the button to wrap the text (you'll have to use a QToolButton), or something along those lines.